F1/F1/Login/LoginViewController.swift

```diff
@@ -9,32 +9,27 @@ import UIKit
 
 class LoginViewController: UIViewController {
 
+    // MARK: IBOutlets
     @IBOutlet weak var username: UITextField!
     @IBOutlet weak var password: UITextField!
-    @IBOutlet weak var errorMsg: UILabel!
+    @IBOutlet weak var errorMessage: UILabel!
+
+    // MARK: Variables
     private lazy var viewModel = LoginViewModel()
```

Comment:
viewModel doesn't take in any delegates or repos so can remove lazy keyword

Reply:
Added a delegate, so no need to remove anymore 😄